File: Model/src/main/java/org/example/Office.java
package org.example;

public class Office extends Entity<Long> {
    private Participant participant;
    private Event event;

    public Office(Participant participant, Event event) {
        this.participant = participant;
        this.event = event;
    }

    public Participant getParticipant() {
        return this.participant;
    }

    public Event getEvent() {
        return this.event;
    }

    public void setParticipant(Participant participant) {
        this.participant = participant;
    }

    public void setEvent(Event event) {
        this.event = event;
    }

    @Override
    public String toString() {
        return "Office{" + super.toString() +
                " participant=" + (participant != null ? participant : "No participant") +
                ", event=" + (event != null ? event : "No event") +
                '}';
    }
}

File: Model/Office.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace mpp_proiect_csharp_DianaGliga11.Model
{
    [Serializable]
    public class Office : Entity<long>
    {
        public Participant Participant { get; set; }
        public Event Event { get; set; }

        public Office(Participant participants, Event events)
        {   
            Participant = participants;
            Event = events;
        }

        public override string ToString()
        {
            return $"ID={Id}, Participants={Participant}, Events={Event}";
        }
    }
}


